src/App.js

```diff
@@ -23,10 +26,40 @@ import ShoeTwo from './lib/components/Ecommerce/slider/images/shoe2.jpg'
 import ShoeThree from './lib/components/Ecommerce/slider/images/shoe3.jpg'
 import ShoeFour from './lib/components/Ecommerce/slider/images/shoe4.jpg'
 import ShoeFive from './lib/components/Ecommerce/slider/images/shoe5.jpg'
+import ShirtOne from './lib/components/Ecommerce/Customcolorcard/Images/one.jpg'
+import ShirtTwo from './lib/components/Ecommerce/Customcolorcard/Images/two.jpg'
+import ShirtThree from './lib/components/Ecommerce/Customcolorcard/Images/three.jpg'
+import ShirtFour from './lib/components/Ecommerce/Customcolorcard/Images/four.jpg'
+import ShirtFive from './lib/components/Ecommerce/Customcolorcard/Images/five.jpg'
+import BangOne from './lib/components/Ecommerce/Customcolorcard/Images/red.jpg'
+import BangTwo from './lib/components/Ecommerce/Customcolorcard/Images/purple.jpg'
+import BangThree from './lib/components/Ecommerce/Customcolorcard/Images/gold.jpg'
+import BangFour from './lib/components/Ecommerce/Customcolorcard/Images/white.jpg'
+import BangFive from './lib/components/Ecommerce/Customcolorcard/Images/multi.jpg'
+import homeone from './lib/components/Ecommerce/PreviewItem/Images/one.jpg'
+import honetwo from './lib/components/Ecommerce/PreviewItem/Images/two.jpg'
+import homethree from './lib/components/Ecommerce/PreviewItem/Images/three.jpg'
+import homefour from './lib/components/Ecommerce/PreviewItem/Images/four.jpg'
+import one from './lib/components/Ecommerce/Topbrand/images/1.png'
```

Comment:
No need to change this now. Ana usually naming romba important, vera yaarachu onnoda code paatha avangaluku puriyaathu enna nu. 

Coding tasks la lam take care of this (it's a good coding practise & standard)